Cubical/Foundations/Prelude.agda

```diff
@@ -286,12 +312,12 @@ module _ (P : ∀ y → x ≡ y → Type ℓ') (d : P x refl) where
 -- Converting to and from a PathP
 
 module _ {A : I → Type ℓ} {x : A i0} {y : A i1} where
-  toPathP : transp (\ i → A i) i0 x ≡ y → PathP A x y
+  toPathP : transp A i0 x ≡ y → PathP A x y
```

Comment:
Oh, no need to eta-expand anymore? @Saizan

Reply:
It's necessary to convert between lines and paths, but that's not happening here. Though it would need to expanded if I switch it to `transport`, since that takes a path. (Still it might be conceptually better to use `transport`?)

Reply:
Yeah, maybe nicer with transport?